<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_pause_menu</class>
 <widget class="QWidget" name="widget_pause_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/frame/resources/frame/pause_menu.png</pixmap>
   </property>
  </widget>
  <widget class="IconButton" name="back_to_game">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background: transparent;       /* 背景透明 */
        border: none;                  /* 无边框 */
    }


</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/button/resources/button/button_.png</normaloff>:/button/resources/button/button_.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="IconButton" name="back_to_menu">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background: transparent;       /* 背景透明 */
        border: none;                  /* 无边框 */
    }


</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/button/resources/button/button_.png</normaloff>:/button/resources/button/button_.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="IconButton" name="game_pause">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>320</width>
     <height>180</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background: transparent;       /* 背景透明 */
        border: none;                  /* 无边框 */
    }


</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/background/resources/background/label_pause.png</normaloff>:/background/resources/background/label_pause.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>320</width>
     <height>180</height>
    </size>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IconButton</class>
   <extends>QPushButton</extends>
   <header>IconButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
